estate/models/estate_property.py

```diff
@@ -0,0 +1,41 @@
+from odoo import fields, models
+import datetime
+from dateutil.relativedelta import relativedelta
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "this is the estate property model"
+    name = fields.Char('Title', required=True)
```

Comment:
```suggestion
    _description = "this is the estate property model"
    
    name = fields.Char('Title', required=True)
```
by convention, the fields are separated by a space from the _name and _description